### PR TITLE
sqllogictest: Fix the Nightly SLT test

### PR DESCRIPTION
**Pull request by philip-stoev**
make a dedicated workflow in test/sqllogictest/mzcompose.yml
the way one existed for the sql-fast test

### CODE REVIEW

File: ci/slt/pipeline.yml [hunk 1]
```diff
@@ -8,7 +8,8 @@
 # by the Apache License, Version 2.0.
 
 steps:
-  - command: ci/slt/slt.sh
```
Comment (philip-stoev) [inline ci/slt/pipeline.yml:11]: @benesch I am not 100% certain about this part. How can I trigger to see if buildkite is happy? Previously you found a way to run nightly against a commit from a personal Materialize fork, but I am not sure how you did it.

Comment (philip-stoev) [reply ci/slt/pipeline.yml:11]: Nevermind, I think I figured it out -- opening  a pull request seems to do something that causes the git sha from the fork to become available for checkout from the main Materialize repository ... so the "New Build" button in the Buildkite UI can be used to build that sha.

Comment (philip-stoev) [reply ci/slt/pipeline.yml:11]: It is green, so I guess it is all good

https://buildkite.com/materialize/sql-logic-tests/builds/4972#11242dcb-7eef-40c7-91d2-61fcaaabbb63

Comment (benesch) [reply ci/slt/pipeline.yml:11]: Yep, that's exactly right!

**Review by benesch — APPROVED** [review]

**Pull request merged**